README

```diff
@@ -156,21 +156,47 @@ REQUIREMENTS:
         dracut (optional)
         PolicyKit (optional)
 
-        When building from git, the following tools are needed:
+        Two build systems are supported: meson + ninja and autotools + make.
+
+        The following tools are needed with both systems:
 
         pkg-config
-        docbook-xsl
-        xsltproc
+        gperf >= 3.1
+        docbook-xsl (optional, required for documentation)
+        xsltproc    (optional, required for documentation)
+        python-lxml (optional, required to build the indices)
+
+        When building with meson, python and ninja-build are required.
+
+        To build in directory build/:
+          meson build/ && ninja-build -C build
```

Comment:
Looks like Fedora is the odd one here, naming the binary ninja-build. Afaics, other distros name the binary ninja, as intendend by upstream. Could we please use ninja instead of ninja-build here and kindly push Fedora to rename the ninja-build binary, see https://bugzilla.redhat.com/show_bug.cgi?id=1166135